src/Atoms/HomeElements/DarKMode.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';

```

Comment:
I did not get the idea of this DarkMode class.

The idea of using styled components is to pass variables to change the colors, letters, or any other thing you need. You don't need to have one style per color.

You can read a little more of this in the styled components page: https://styled-components.com/docs/basics